example/lib/peripheral_list/components/peripheral_row_view.dart

```diff
@@ -66,8 +75,6 @@ class PeripheralRowView extends StatelessWidget {
   }
 
   Color _colorForRssi(int rssi) {
-    return rssi > -60
-        ? Colors.blue
-        : rssi > -90 ? Colors.orange : Colors.red;
+    return rssi > -60 ? Colors.blue : rssi > -90 ? Colors.orange : Colors.red;
```

Comment:
Would be more readable in an if/else cascade.

Reply:
Refactored to if/else cascade.